packages/calcite-components/src/components/block/block.stories.ts

```diff
@@ -1,20 +1,22 @@
+import { BlockSection } from "../block-section/block-section";
 import { boolean } from "../../../.storybook/utils";
 import { placeholderImage } from "../../../.storybook/placeholderImage";
 import { html } from "../../../support/formatting";
 import { ATTRIBUTES } from "../../../.storybook/resources";
+import { Block } from "./block";
 const { toggleDisplay } = ATTRIBUTES;
 
-interface BlockArgs {
-  heading: string;
-  description: string;
-  blockOpen: boolean;
-  collapsible: boolean;
-  loading: boolean;
-  disabled: boolean;
-  headingLevel: number;
+interface BlockStoryArgs {
```

Comment:
I _think_ this can be simplified a bit by using `Pick`:

```ts
interface BlockStoryArgs extends 
  Pick<Block, "heading" | "description" | "open" | "collapsible" | "loading" | "disabled" | "headingLevel">, 
  Pick<BlockSection, "open" | "toggleDisplay"> {
  text: string;
}
```

Applies to similar args types.

Reply:
Both Block and BlockSection have an `open` prop, I had to declare them as `blockOpen: Block["open"];` and `sectionOpen: BlockSection["open"];`  to be able to use them in storybook's ui. If I use the Pick format the `open` props conflict and storybook's ui doesn't work.

I could pull Block's open prop with Pick and separately declare BlockSection's open prop as `sectionOpen` to avoid conflicts. Please let me know If this is okay, I can follow that same pattern with all other story interfaces.

`interface BlockStoryArgs
  extends Pick<Block, "heading" | "description" | "open" | "collapsible" | "loading" | "disabled" | "headingLevel">,
    Pick<BlockSection, "toggleDisplay"> {
  sectionOpen: boolean;
  text: string;
}`

Reply:
Thanks for clarifying! Your suggestion looks good. 

You could also use `BlockSection`'s type to keep the renamed prop in sync and also make it clear that it's coming from section.

```tsx
 extends Pick<Block, "heading" | "description" | "open" | "collapsible" | "loading" | "disabled" | "headingLevel">,
    Pick<BlockSection, "toggleDisplay"> {
  sectionOpen: BlockSection["open"];
  text: string;
}
```